src/pages/search/map.tsx

```diff
@@ -146,7 +159,15 @@ export default function MapSearch() {
           <SearchPageSearchForm showTitle={false} searchRoute="/search/map" />
         }
       />
-      <MapView items={searchResultItems} />
+      {showVenueFocusedMap ? (
+        <MapView
+          key="focusedMap"
+          items={searchResultItems}
+          focusedItemId={venueId}
+        />
+      ) : (
+        <MapView items={searchResultItems} focusedItemId={venueId} />
+      )}
```

Comment:
What's happening here? To the naked eye it looks like the conditional isn't needed here.

Reply:
Maybe this has something to do with forcing the map to re-render?